sass/themes/shop/2019/_shop19-components.scss

```diff
@@ -0,0 +1,52 @@
+// SR components
```

Comment:
CR components ... sorry was going for the new line comment really.. :p

Reply:
_jeez_